### PR TITLE
BUGFIX: SD-661 Scatter/Bubble chart - Name toggle is enabled

### PR DESCRIPTION
**Pull request by naibinh**
---

[Check PR owner responsibilities](https://confluence.intgdc.com/display/Development/Code-reviews#Code-reviews-Ownerresponsibilities)

Supported PR commands:

Command | Description
--- | ---
`extended test - examples` | Live examples tests
`extended test - storybook` | Storybook screenshot tests

See more [options](https://confluence.intgdc.com/display/kbhr/How+to+work+with+git+and+Github#HowtoworkwithgitandGithub-Extendedchecks).

# PR Checklist

- [x] Verify code changes ([Checklist](https://confluence.intgdc.com/display/Development/Code-reviews+checklist), [Best practices](https://confluence.intgdc.com/display/Development/Code-reviews+best+practices))
- [x] [Verify pull-request formalities](https://confluence.intgdc.com/display/Development/Code-reviews)
- [x] Change was tested by using [gdc-dev-release](https://confluence.intgdc.com/display/~tomas.vojtasek/Private+NPM) in [gdc-analytical-designer](https://github.com/gooddata/gdc-analytical-designer) and [gdc-dashboards](https://github.com/gooddata/gdc-dashboards) (if applicable)
- [x] Migration guide (for major changes) is mentioned in [CHANGELOG.md](../blob/master/CHANGELOG.md).
- [x] Successful `extended test - examples`
- [x] Successful `extended test - storybook`
- [x] Checked yarn.lock consistency (no dep. duplicities especially Goodstrap)

# Related PRs
- gdc-analytical-designer: https://github.com/gooddata/gdc-analytical-designer/pull/2517
- gdc-dashboards: https://github.com/gooddata/gdc-dashboards/pull/2327

# Related Jira tasks
- SD-661: https://jira.intgdc.com/browse/SD-661

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/4921/ : SUCCESS in 11m 04s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/256/ : SUCCESS in 6s

**Comment by naibinh** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/4922/ : SUCCESS in 6m 12s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/257/ : SUCCESS in 6s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/2591/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/258/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/2000/ : SUCCESS in 13m 33s

**Comment by naibinh** [comment]
extended test - examples

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/259/ : SUCCESS in 12s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/2001/ : SUCCESS in 25m 31s

File: src/internal/components/configurationPanels/tests/BaseChartConfigurationPanel.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+            expect(xAxisSection.props().disabled).toEqual(false);
+
+            const yAxisSection = axisSections.at(1);
+            expect(yAxisSection.props().disabled).toEqual(false);
+
+            const secondaryYAxisSection = axisSections.at(2);
+            expect(secondaryYAxisSection.props().disabled).toEqual(false);
+        });
+
+        it("should render configuration panel with enable X axis name section and disable Y axis name section in single axis chart", () => {
```
Comment (ptquang86) [inline src/internal/components/configurationPanels/tests/BaseChartConfigurationPanel.spec.tsx]: is it `... enabled ... disabled ...`?
also apply for other tests :)

Comment (naibinh) [reply src/internal/components/configurationPanels/tests/BaseChartConfigurationPanel.spec.tsx]: Fixed

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/4923/ : SUCCESS in 8m 05s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/260/ : SUCCESS in 9s

File: src/internal/components/configurationPanels/tests/BaseChartConfigurationPanel.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+                                visualizationAttribute: {
+                                    localIdentifier: "viewId",
+                                    displayForm: { uri: "/gdc/md/projectId/obj/1024" },
+                                },
+                            },
+                        ],
+                    },
+                ],
+                visualizationClass: { uri: "/gdc/md/projectId/obj/75535" },
+            };
```
Comment (lanhq) [inline src/internal/components/configurationPanels/tests/BaseChartConfigurationPanel.spec.tsx]: Duplicate bucket items. Should declare some variable and reuse them.

Comment (naibinh) [reply src/internal/components/configurationPanels/tests/BaseChartConfigurationPanel.spec.tsx]: No, I don't think it's really neccessary.

Comment (lanhq) [reply src/internal/components/configurationPanels/tests/BaseChartConfigurationPanel.spec.tsx]: you mean that leave duplication here is better.
it isn't look good to me.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/4954/ : SUCCESS in 4m 40s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/292/ : SUCCESS in 14s

**Comment by lanhq** [comment]
LGTM

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/4955/ : SUCCESS in 9m 10s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/293/ : SUCCESS in 12s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/4956/ : SUCCESS in 3m 53s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/294/ : SUCCESS in 6s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/2606/ : UNSTABLE in 13m 09s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/4957/ : SUCCESS in 5m 42s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1235/ : SUCCESS in 18s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/1626/ : SUCCESS in 21m 18s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/2607/ : SUCCESS in 10m 24s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/4959/ : SUCCESS in 13m 22s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1236/ : SUCCESS in 43s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/1627/ : SUCCESS in 12m 11s

**Pull request merged**